man/bfb-install.8

```diff
@@ -5,7 +5,15 @@ bfb-install \- BFB installing script for BlueField SoC over rshim driver
 
 .SH SYNOPSIS
 .B bfb-install
--b, --bfb <bfb_file> -r, --rshim [<ip:>[:port]:]rshim<N> [-m, --remote-mode <scp|nc|ncpipe>] [-R, --reverse-nc] [-c, --config <config_file>] [-f, --rootfs <rootfs_file>] [-h, --help] [-v, --verbose]
+-b, --bfb <bfb_file> -r, --rshim [<ip:>[:port]:]rshim<N>
```

Comment:
The change tries to split the option into separate lines, but this line has two options. Is it on purpose?

Reply:
These two options are mandatory options, while others are all optional.